httpie/output/formatters/colors.py

```diff
@@ -15,22 +16,32 @@
 from pygments.util import ClassNotFound
 
 from ..lexers.json import EnhancedJsonLexer
+from ..ui.palette import COLOR_PALETTE
 from ...compat import is_windows
 from ...context import Environment
 from ...plugins import FormatterPlugin
 
 
 AUTO_STYLE = 'auto'  # Follows terminal ANSI color styles
 DEFAULT_STYLE = AUTO_STYLE
-SOLARIZED_STYLE = 'solarized'  # Bundled here
+
+# Bundled here
+SOLARIZED_STYLE = 'solarized'
+PIE_STYLES = {
+    'pie',
+    'pie-light',
+    'pie-dark'
+}
```

Comment:
Looks like an enum/constants candidate (we refer to it from multiple places). Maybe it could also go to `palette`.